From 830ec41d30fc972dafa50de5b7220adbe0bfac35 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Wed, 31 May 2017 16:36:17 +0800
Subject: [PATCH 1832/5242] MLK-14985-1 Documentation: i2c: add dt
 documentation for isl29023

commit  43796b321be662e00d8502235f8f103b2771ab50 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dt documentation for intersil ISL29023 ambient light sensor.

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/trivial-devices.txt        |    1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/trivial-devices.txt b/Documentation/devicetree/bindings/trivial-devices.txt
index 6d3e635..3b26a9a4 100644
--- a/Documentation/devicetree/bindings/trivial-devices.txt
+++ b/Documentation/devicetree/bindings/trivial-devices.txt
@@ -55,6 +55,7 @@ epson,rx8010		I2C-BUS INTERFACE REAL TIME CLOCK MODULE
 epson,rx8581		I2C-BUS INTERFACE REAL TIME CLOCK MODULE
 fsl,fxas2100x		FXAS2100X: Gyroscope sensor
 fsl,fxos8700		FXOS8700: Accelerometer + Magnetometer Combo
+fsl,isl29023		ISL29023: Intersil ISL29023 ambient light sensor
 emmicro,em3027		EM Microelectronic EM3027 Real-time Clock
 fsl,mag3110		MAG3110: Xtrinsic High Accuracy, 3D Magnetometer
 fsl,mma7660		MMA7660FC: 3-Axis Orientation/Motion Detection Sensor
-- 
1.7.9.5

